### ISSUE TITLE
Chart accesibility

### ISSUE THREAD

**Issue by JoltCode** [AnswerOverflow/AnswerOverflow#120]
### What is this for?

website

### Feature

As an impaired user, I should be able to utilise the chart components with ease.

### Ideal solution or implementation

Charts component needs to be looked at accessibility wise - it's currently failing a11y tests.

### Alternative solutions or implementations

Currently, a11y tests are disabled for the chart component

### Other context

n/a
